.circleci/config.yml

```diff
@@ -330,6 +341,12 @@ workflows:
           requires:
             - electron-runner-image-push
 
+      - bundle-stats:
+          <<: *not_staging_or_release
+          context: hokusai
```

Comment:
What permissions are actually necessary for this step? This context has access to release to (and read from) our cluster which I imagine isn't necessary. If access to S3 is necessary, there's probably a more restrictive context we can lean on.

Reply:
Right, this just needs to run the build to emit the stats then upload them using the agent + the environment variable that's set in Circle. I, however, don't understand the config or how to use a more restrictive context, in this case.

Reply:
> upload them using the agent + the environment variable that's set in Circle

Which environment variable?

Reply:
`RELATIVE_CI_KEY`

Reply:
So, what context should I use though? I apparently need the AWS creds just to run the build.

Reply:
If you mean the prerequisite builder-image-push step by "the build," I think that already uses the `hokusai` context. Does this step also need AWS creds? If not, I'd create a new `relativeci` org context, add the key there, and specify it here.

Reply:
Just did that. Apparently not:

![](https://capture.static.damonzucconi.com/Screen-Shot-2024-08-16-08-22-11.17-9sasgxVkF8Nd29kWtL0MXuoOpID9rOwRR22stZUibgG4qjaTw5Rxhm28dSNEQq4gm3BSYGIJ0KU9HTbZ16fxji5YSQIQyMg3Uklg.png)

In the interest of moving on I'm going to proceed with the Hokusai context for now.